Dockerfile

```diff
@@ -1,16 +1,15 @@
-# Docker image for the Drone SFTP plugin
-
 FROM alpine:3.3
 
 RUN apk update && \
-    apk add nodejs && \
-    apk add git && \
```

Comment:
As you can see at http://beta.drone.io/drone-plugins/drone-sftp/4 all dependencies are installing fine :)